e2e/test/scenarios/native/native-reproductions.cy.spec.js

```diff
@@ -586,6 +586,7 @@ describe("issue 34330", () => {
 
     // can't use cy.type because it does not simulate the bug
     // Delay needed for React 18. TODO: fix shame
+    cy.wait(50);
```

Comment:
@romeovs I tested locally with x6 throttling without this `cy.wait` and it passed to me. Could you please try to remove it?

if it doesn't work, there should be a popup with `loading...` message, so we can rely on it